src/Gui/InputField.cpp

```diff
@@ -175,12 +186,10 @@ void InputField::updateText(const Base::Quantity& quant)
     setText(QString::fromStdString(txt));
 }
 
-void InputField::resizeEvent(QResizeEvent *)
+void InputField::resizeEvent(QResizeEvent * /*event*/)

```

Comment:
Unrelated? No benefit of this either

Reply:
common practice, is either that or using Q_UNUSED, it avoids the IDE screaming at you, not important

Reply:
I think that we should use `[[maybe_unused]]` in that case as it is the current standard and independent of Qt.